packages/connect/src/api/cardano/cardanoUtils.ts

```diff
@@ -47,6 +47,7 @@ export const prepareCertificates = (certs: CardanoCertificate[]) => {
                 convertedCerts.push({
                     type: cert.type,
                 });
+                // conway certificates not supported by coin-selection lib yet
```

Comment:
maybe we should add TODO to this comment so that someone will notice it in the future?